prov/sockets/src/sock_ep.c

```diff
@@ -584,7 +584,10 @@ static ssize_t sock_tx_size_left(struct fid_ep *ep)
 	fastlock_acquire(&tx_ctx->wlock);
 	num_left = rbavail(&tx_ctx->rb)/SOCK_EP_TX_ENTRY_SZ;
 	fastlock_release(&tx_ctx->wlock);
-	return num_left;
+	if (num_left > SOCK_EP_TX_SZ)
```

Comment:
> Yes, when we are allocating ring buffer for tx_ctx, we roundup the size to the power of two.

There's something else going on because before I was getting `368`, which isn't a power of two.